test/support/utils_invoke.h

```diff
@@ -195,17 +216,15 @@ inline void unsupported_types_notifier(const sycl::device& device)
 template <::std::size_t CallNumber = 0>
 struct invoke_on_all_hetero_policies
 {
-    sycl::queue queue;
-
-    invoke_on_all_hetero_policies(sycl::queue _queue = get_test_queue())
-        : queue(_queue)
-    {
-    }
-
     template <typename Op, typename... Args>
     void
     operator()(Op op, Args&&... rest)
     {
+        using kernel_name = unique_kernel_name<Op, CallNumber>;
+        auto my_policy = get_dpcpp_test_policy<kernel_name>();
```

Comment:
This one just becomes `get_dpcpp_test_policy</*call_id=*/ 0,kernel_name>();` which is not so bad

Reply:
Fixed